pyproject.toml

```diff
@@ -215,7 +219,7 @@ exclude = [
 ]
 
 [tool.uv.workspace]
-members = ["conformance", "example", "noextras"]
```

Comment:
I think the main reason I had it as a workspace member was I didn't think pytest would behave well otherwise, but apparently it does.

I also have a longish term idea of running tests with more app servers, HTTP/1 uvicorn, WSGI gunicorn, etc. It's probably fine to add them all in dev-dependencies here but what do you think?

Reply:
I'm fine with keeping them around if there's a good technical reason, honestly I just didn't grok all of it at first :). Might be less of a big deal if renovate helps with keeping the pins in sync.